### DISCUSSION TITLE
Docker installation- Error installing k2

### DISCUSSION THREAD

**Discussion by scramble40** [NVIDIA-NeMo/NeMo#4272] [Q&A] [answered]
Hi, while composing with the Dockerfile I ran into the following error:

executor failed running [/bin/sh -c /bin/bash /tmp/nemo/scripts/speech_recognition/k2/setup.sh; exit 0]: exit code: 137

Any suggestions on how to solve this, or should I just comment this line in the file and try to setup the container without it?

**Comment by titu1994** [comment, marked as answer]
You can comment if out. It's not needed for most of ASR.

**Reply by scramble40** [reply]
Great, thanks.

**Reply by ahkarami** [reply]
@titu1994 
Hi,
It seems that K2 needs PyTorch 1.7 and NeMo needs PyTorch 1.10 or above, and so they have conflict with each other. Is there any possibility to address this issue?

**Reply by titu1994** [reply]
@GNroy we need comment out K2 until this is fixed on their end ?
